src/python/gudhi/off_reader.pyx

```diff
@@ -32,6 +34,6 @@ def read_points_from_off_file(off_file=''):
         if os.path.isfile(off_file):
             return read_points_from_OFF_file(off_file.encode('utf-8'))
         else:
-            print("file " + off_file + " not found.")
+            print("file " + off_file + " not found.", file=sys.stderr)
```

Comment:
Would it make sense to raise FileNotFoundError (or other) instead of printing a warning? I don't know if that should be done here or in read_points_from_OFF_file or further down the chain. That can be a separate issue/PR.